jdaviz/configs/cubeviz/plugins/tests/test_cubeviz_unit_conversion.py

```diff
@@ -35,19 +35,92 @@ def test_basic_unit_conversions(cubeviz_helper, angle_unit):
 
     # load cube with flux units of MJy
     w, wcs_dict = cubeviz_wcs_dict()
-    flux = np.zeros((3, 4, 5), dtype=np.float32)
-    cube = Spectrum1D(flux=flux * u.MJy / angle_unit, wcs=w, meta=wcs_dict)
+    flux = np.ones((3, 4, 5), dtype=np.float32)
+    cube = Spectrum1D(flux=flux * (u.MJy / angle_unit), wcs=w, meta=wcs_dict)
     cubeviz_helper.load_data(cube, data_label="test")
+    viewer = cubeviz_helper.app.get_viewer("spectrum-viewer")
 
     # get all available flux units for translation. Since cube is loaded
     # in Jy, this will be all items in 'spectral_and_photon_flux_density_units'
 
     uc_plg = cubeviz_helper.plugins['Unit Conversion']
+    ap_plg = cubeviz_helper.plugins["Aperture Photometry"]._obj
+    label_mouseover = cubeviz_helper.app.session.application._tools['g-coords-info']
+
+    cubeviz_helper.load_regions(RectanglePixelRegion(PixCoord(1, 1), 1, 1))
+    ap_plg.background_selected = "Subset 1"
 
     for flux_unit in SPEC_PHOTON_FLUX_DENSITY_UNITS:
+        if flux_unit == 'MJy':
```

Comment:
Note: I calculated these by hand, so should be good enough for this test. Had to bump the rtol up a little and not sure why;  maybe there are extra funky calculations in the logic route that changes floating point precision slightly, but not enough for concern, I don't think.